src/main/java/ch/nexusnet/postmanager/aspect/LoggingAspect.java

```diff
@@ -0,0 +1,40 @@
+package ch.nexusnet.postmanager.aspect;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+public class LoggingAspect {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    // Pointcut defining where the logger gets used
+    @Pointcut("within(@org.springframework.stereotype.Repository *)" +
+            " || within(@org.springframework.stereotype.Service *)" +
+            " || within(@org.springframework.web.bind.annotation.RestController *)")
```

Comment:
If this Aspect is meant to be used throughout the post-manager service, you should also consider extending the point cut with an `@Component` annotation, since you are using them as well.  (Not mandatory)

Reply:
Good catch. I will add it. Thanks!